OpenOversight/app/templates/base.html

```diff
@@ -38,14 +38,16 @@
           rel="stylesheet">
     <!-- jQuery -->
     <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
-    <!-- Bootstrap -->
-    {{ bootstrap.load_css() }}
-    {{ bootstrap.load_js() }}
+    <!-- Custom styles for this template -->
+    <link href="{{ url_for('static', filename='css/fonts.css') }}"
+          rel="stylesheet">
+    <link href="{{ url_for('static', filename='css/openoversight.css') }}"
```

Comment:
I guess we weren't really overriding the bootstrap styles, but this does seem like an order change we should have had in place a long time ago.